supportedBackends/htcondor/src/main/scala/cromwell/backend/impl/htcondor/HtCondorJobExecutionActor.scala

```diff
@@ -136,9 +152,23 @@ class HtCondorJobExecutionActor(override val jobDescriptor: BackendJobDescriptor
   /**
     * Abort a running job.
     */
-  override def abort(): Unit = throw new UnsupportedOperationException("HtCondorBackend currently doesn't support aborting jobs.")
+  override def abort(): Unit = {
+    // Nothing to do in case `condorJobId` is not defined
+    condorJobId foreach { id =>
+      log.info("{}: Aborting job [{}:{}].", tag, jobDescriptor.key.tag, id)
+      val abortProcess = new HtCondorProcess
+      val argv = Seq(HtCondorCommands.Remove, id)
+      val process = abortProcess.externalProcess(argv)
+      val exitVal = process.exitValue()
```

Comment:
IIRC the semantics of `abort()` are:
* This will trigger an abort of the underlying code but not do anything synchronously in the method
* Nothing is returned and no reply messages are sent out directly by this method
* When the `abort()` causes the underlying job to stop, the BJEA will notice this as part of its ongoing monitoring of the job and can then reply to its parent with the `AbortedResponse`